portfolio_style.css

```diff
@@ -0,0 +1,155 @@
+@import url('https://fonts.googleapis.com/css?family=Charm|Fredericka+the+Great');
+
+html {
+	font-family: "Fredericka the Great";
+	color: #5f6161;
+	font-size: 10pt;
+	
+}
+
+*, *::before, *::after {
+	margin: 0;
+	padding: 0;
+	box-sizing: border-box;
+}
+	
+body {
+	background-color: white;
+}
+
+a {
+	color: white;
+	text-align: center;
+}
+
+a:hover {
+	color: #e00dbd
+}
+
+a:focus{
+	color:#e00dbd;
+}
+
+p {
+	text-align: center;
+}
+
+h1, h2 {
+	color: #b5b0b3;
+	text-align: center;
+}
+
+h1 {
+	font-size: 50pt;
+}
+
+.nav_bar {
+	display: flex;
+	flex-direction: row;
+	justify-content: space-between;
+	background-color: #e69ccb;
+	color: white;
+	height: 50px;
+	padding: 10px;
+	width: 100%;
+
+}
+
+.section {
+	display: flex;
+	flex-direction: row;
+	flex-wrap: nowrap;
+	padding-left: 50px;
+	padding-right: 50px;
+
+}
+
+.welcome {	
+	background-color: #f7daed;
+	font-size: 30pt;
+	padding-bottom: 200px;
+	height: 90%;
+}
+
+.contact {
+	background-color: #f7daed;
+	font-size: 20pt;
+	padding-bottom: 200px;
+	padding-top: 100px;
```

Comment:
use rem

Reply:
I changed the font to rem, but the padding I left it in px